### ISSUE TITLE
rebuild docker image and pin discord.py version

### ISSUE THREAD

**Issue by classabbyamp** [0x5c/minibot#20]
see miaowware/qrm2#239

**Comment by classabbyamp** [comment]
```patch
index 844f49a..51d83bc 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1 +1 @@
-discord.py
+discord.py==1.3.4
```

**Comment by 0x5c** [comment]
Fixed in 6fde836724f82c2dc60227b5297dd7e6b45a442c
